### PR TITLE
Prepare Android for v1.3.0

### PR DESCRIPTION
**Pull request by william-ct**
Add Increment/ Decrement methods.
Add InApp control methods.
Deprecate profileGetCleverTapID() and profileGetCleverTapAttributionIdentifier().
Add a new public method getCleverTapID() as an alternative to above-deprecated methods.
Update version to 1.3.0

### CODE REVIEW

**Comment by darshanclevertap** [comment]
@william-ct You need to update the CHANGELOG and README as well

**Review by Aditi3 — COMMENTED** [review]
Let's move to version `1.3.0` instead of `1.2.4`. Since the release includes feature(s)

**Review by Aditi3 — APPROVED** [review]
LGTM 👍🏼

File: lib/clevertap_plugin.dart [hunk 1]
```diff
@@ -426,17 +426,24 @@ class CleverTapPlugin {
         'setLocation', {'latitude': latitude, 'longitude': longitude});
   }
 
+  @Deprecated("This method is deprecated since v1.2.4. Use getCleverTapID() instead")
```
Comment (Aditi3) [inline lib/clevertap_plugin.dart:429]: Should be `This method is deprecated since v1.3.0. Use getCleverTapID() instead` @william-ct

**Comment by Aditi3** [comment]
Good to merge from my side @william-ct

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 ## CHANGE LOG
 
+### Version 1.3.0 (September 7, 2021)
+* Adds public methods for suspending/discarding & resuming InApp Notifications
```
Comment (darshanclevertap) [inline CHANGELOG.md:4]: @william-ct change this to - suspending, discarding & resuming instead of suspending/discarding

**Review by darshanclevertap — CHANGES_REQUESTED** [review]
@william-ct Just one change in the CHANGELOG, the rest looks good to me 👍🏾

File: CHANGELOG.md (hunk 2 above)
Comment (william-ct) [reply CHANGELOG.md]: Done👍

**Review by darshanclevertap — APPROVED** [review]
LGTM 👍🏾

**Pull request merged**